pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "search"
+version = "0.1.0"
```

Comment:
Can we not version it here?

Reply:
I don't understand the question given a version number is supplied. What are you expecting instead?

Reply:
Does Poetry require a `version` field? It's better not to have it in the codebase (instead as Git/Docker tags).

Reply:
It seems it is required currently and will not install packages without one. When creating a python package this is required in `setup.py` but we are only using it for installing packages so it's not necessary for this repo. I can create an issue for this on the `poetry` repo but I don't think it will be resolved soon. We can either ignore this for now or keep it up to date when merging to master. Suggestions welcome.

Reply:
Shame, we should probably just keep it as a fake value rather than worrying about maintaining it (assume it's not going to have any value?).

Reply:
No this shouldn't have any implications for this project other than being a little confusing. I'll see what the author suggests but leave it with the current version for now.